pkgs/development/libraries/webkitgtk/default.nix

```diff
@@ -78,6 +83,16 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-1VMvqITJQ9xI8ZEUc91mOrpAejs1yqewS6wUGbQeWQg=";
   };
 
+  # Cocoa headers are not packaged in the tarball
```

Comment:
Could you open a bug upstream so that they are added? Having to update multiple hashes when updating is quite annoying since we can no longer simply use `update-source-version`. Alternatively, you could add an update script ([example](https://github.com/NixOS/nixpkgs/blob/2839e5c7fad06c731008884cac9325f596363432/pkgs/tools/graphics/gmic/default.nix#L80-L99)).

Reply:
Opened Bug 243759. Or we could simply fetch the tarball from GitHub, probably.

Reply:
Updated the patch so that Cocoa headers are no longer needed.